Client.xcodeproj/xcshareddata/xcschemes/Fennec_Enterprise_XCUITests.xcscheme

```diff
@@ -167,7 +167,7 @@
       </BuildActionEntries>
    </BuildAction>
    <TestAction
-      buildConfiguration = "Fennec"
+      buildConfiguration = "Fennec_Enterprise"
```

Comment:
Not sure why this change appears here but it should not be this Fennec_Enterprise configuration for Fennec_Enterprise_XCUITests... please rever this change